### PR TITLE
Performed some code climate improvements for the Company module

### PR DESCRIPTION
**Pull request by Koen1999**
This is in preperation for the SEP group who will work on the Company module.

### CODE REVIEW

**Comment by Koen1999** [comment]
This fixes 24 codeclimate issues

File: module/Company/src/Company/Service/Company.php [hunk 1]
```diff
@@ -975,8 +975,7 @@ public function getEditableJobsByLanguageNeutralId($languageNeutralId)
                 $translator->translate('You are not allowed to edit jobs')
             );
         }
-        $res = $this->getJobMapper()->findJob(['languageNeutralId' => $languageNeutralId]);
-        return $res;
+        return $this->getJobMapper()->findJob(['languageNeutralId' => $languageNeutralId]);
```
Comment (tomudding) [inline module/Company/src/Company/Service/Company.php:978]: Empty line.

File: module/Company/view/company/admin/edit-company.phtml [hunk 2]
```diff
@@ -71,64 +71,63 @@ $submit->setAttribute('class', 'btn btn-primary pull-right');
...
+        <th>
+            <?= $this->translate('Expiration date'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Jobs / Type'); ?>
+        </th>
+        <th>
+            <?= $this->translate('Actions'); ?>
+        </th>
+    </tr>
```
Comment (tomudding) [inline module/Company/view/company/admin/edit-company.phtml:90]: I don't see why we would have to remove one level of indentation here.

File: module/Company/view/company/company/list.phtml [hunk 3]
```diff
@@ -37,8 +37,8 @@ $this->headTitle($this->translate('Companies'));
                             <a href="<?= $this->url('company/companyItem', // Route url
                                 ['slugCompanyName' => $company->getSlugName()], // Url parameters
                                 [], // routeOptions
-                                true); // Reuse matched parameters  ?>"
-                                class="list-group-item"><?= $company->getName() ?></a>
+                                true); // Reuse matched parameters   ?>"
```
Comment (tomudding) [inline module/Company/view/company/company/list.phtml:40]: Don't think another ` ` is needed here.

**Review by tomudding — CHANGES_REQUESTED** [review]

File: module/Company/view/company/admin/edit-company.phtml (hunk 2 above)
Comment (Koen1999) [reply module/Company/view/company/admin/edit-company.phtml:90]: I agree. PhpStorm thought this was neccesarry to do somehow. But looking at the opening and closing tags it should really have one more level.

**Review by tomudding — DISMISSED** [review]

**Review by tomudding — APPROVED** [review]

**Pull request merged**